File: src/paths.ts
import {homedir} from 'node:os';
import path from 'node:path';

export const CACHE_DIR = path.join(homedir(), '.cache', 'how');
export const OPTIONS_PATH = path.join(CACHE_DIR, 'options.json');
export const TLDR_PATH = path.join(CACHE_DIR, 'tldr');
export const GLOW_PATH = path.join(CACHE_DIR, 'glow');

File: internal/paths.go
package internal

import (
	"os"
	"path/filepath"
)

var homeDir, _ = os.UserHomeDir()

var CacheDir = filepath.Join(homeDir, ".cache", "how")
var OptionsPath = filepath.Join(CacheDir, "options.json")
var TldrPath = filepath.Join(CacheDir, "tldr")
